README.md

```diff
@@ -40,6 +32,47 @@ mvn clean install
 
 This will build the artifacts after validating all checks including unit tests and place it in the deployer/target directory.
 
+
+## Installation
+
+* Build the project.
+* Login cf and select the space that Ocopea will deploy to.
+* Ensure a Postgres service and a MySQL service named 'crb-mysql' exist in the selected space. Run
```

Comment:
Do we need crb-mysql for shpancrb? Since we don't yet support pcf-fs-crb I would not add this requirement.

Reply:
Fixed the issue.